packages/techdocs-common/src/stages/publish/helpers.test.ts

```diff
@@ -73,3 +77,17 @@ describe('getFileTreeRecursively', () => {
     expect(fileList).toContain(path.resolve(root, 'subDirA/file2'));
   });
 });
+
+describe('lowerCaseEntityTripletInStoragePath', () => {
+  it('returns lower-cased entity triplet path', () => {
```

Comment:
would it be good to test if you get paths in unexpected formats? like not having a triplet? Eg: /default/component/IMAGE.png ?

Reply:
Hey @soapraj can you review it again, please?